src/screens/ChannelListScreen/index.tsx

```diff
@@ -53,12 +71,26 @@ const ChannelListScreen = () => {
   };
 
   return (
-    <FlatList
-      data={channels}
-      keyExtractor={(item) => item.id}
-      scrollEnabled={false}
-      renderItem={renderChannelItem}
-    />
+    <>
+      <StatusBar translucent={false} />
+      <View style={{height: 52}}>
+        <Search route={route} onPress={goToContactScreen} />
+      </View>
+
+      <FlatList
+        data={channels}
+        keyExtractor={(item) => item.id}
+        ListHeaderComponent={
+          <ChannelListHeaderItem
+            name={`@${profile?.username}`}
+            picture={profile?.profile_pic_path}
+            type="SIGNED"
+            testID="horizontal-tab-0"
+          />
+        }
+        renderItem={renderChannelItem}
+      />
+    </>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a `StatusBar` component and wraps the `Search` component and `FlatList` component inside a fragment. The `FlatList` now includes a `ListHeaderComponent` prop which renders a `ChannelListHeaderItem`. This seems like a good change as it enhances the UI of the channel list screen. However, make sure that the `profile` object is always defined before trying to access its properties to avoid potential errors.

```diff
+ <>
+   <StatusBar translucent={false} />
+   <View style={{height: 52}}>
+     <Search route={route} onPress={goToContactScreen} />
+   </View>
+   <FlatList
+     data={channels}
+     keyExtractor={(item) => item.id}
+     ListHeaderComponent={
+       <ChannelListHeaderItem
+         name={`@${profile?.username}`}
+         picture={profile?.profile_pic_path}
+         type="SIGNED"
+         testID="horizontal-tab-0"
+       />
+     }
+     renderItem={renderChannelItem}
+   />
+ </>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->